mlflow/tracking/_model_registry/fluent.py

```diff
@@ -21,6 +21,36 @@ def register_model(model_uri, name):
                  automatically.
     :return: Single :py:class:`mlflow.entities.model_registry.ModelVersion` object created by
              backend.
+
+    .. code-block:: python
+        :caption: Example
+
+        from pprint import pprint
+        import mlflow
+
+        local_store_uri = "sqlite:///api_mlruns.db"
+        mlflow.set_tracking_uri(local_store_uri)
+
+        # Set an existing run_id
+        run_id = "acd04001d9874ce5956f701583596cbc"
+        model_uri = "runs:/{}".format(run_id)
+
+        # Register the model
+        mv = mlflow.register_model(model_uri, "RandomForestRegressionModel")
+        pprint("Registered Model Version Info: {}".format(mv))
```

Comment:
Can we make this example runnable using a dummy model?

Reply:
Yes, we can. How about something like this:

```
from pprint import pprint
import mlflow.sklearn
from sklearn.ensemble import RandomForestRegressor

local_store_uri = "sqlite:///api_mlruns.db"
mlflow.set_tracking_uri(local_store_uri)

 # Log MLflow entities
 with mlflow.start_run(run_name="My Runs") as run:
        params = {"n_estimators": 3, "random_state": 42}
        sk_learn_rfr = RandomForestRegressor(params)
        mlflow.log_params(params)
        mlflow.sklearn.log_model(sk_learn_rfr, artifact_path="sklearn-model")

    model_uri = "runs:/{}".format(run.info.run_id)
    mv = mlflow.register_model(model_uri, "RandomForestRegressionModel")
    pprint("Registered Model Version Info: {}".format(mv))

('Registered Model Version Info: <ModelVersion: '
 "creation_timestamp=1599848876966, current_stage='None', description=None, "
 "last_updated_timestamp=1599848876966, name='RandomForestRegressionModel', "
 "run_id='0b0f0bb2a6ec4faca80506dadd0ed7f1', run_link=None, "
 "source='./mlruns/0/0b0f0bb2a6ec4faca80506dadd0ed7f1/artifacts', "
 "status='READY', status_message=None, tags={}, user_id=None, version=1>")
```

Reply:
```
from pprint import pprint
import mlflow

mlflow.set_tracking_uri("sqlite:////tmp/mlruns.db")


class MyModel(mlflow.pyfunc.PythonModel):
    pass


with mlflow.start_run() as run:
    mlflow.pyfunc.log_model("model", python_model=MyModel())

model_uri = "runs:/{}".format(run.info.run_id)

mv = mlflow.register_model(model_uri, "my-model")

pprint("Registered Model Version Info: {}".format(mv))

```

Reply:
IMHO, we may confuse the beginner with the PyFuncModel. Let's leave this to pyfunc flavor examples for later model flavor code and customized pyfunc models. That area seems to be confusing for many beginners.

Reply:
Makes sense. Can we use a sklearn model instead which I think most users are familiar with?

Reply:
Sorry I missed your code example

Reply:
I’m using a sklearn model, unless you mean use something simpler than RandomForestRegressor.

Reply:
Is this model uri: `model_uri = "runs:/{}".format(run.info.run_id)` (in your code example) really correct?

Reply:
Ah, artifact_path is missing. [Fixed](https://github.com/dmatrix/mlflow-tests/blob/master/py/mlflow/apis/fluent/register_model.py)!